src/animation/use-animate-prop.ts

```diff
@@ -87,7 +96,17 @@ export function useAnimateProp(
                     finalTarget[key]
                 )
 
-                if (isValidValue && (valueHasUpdated || shouldAnimateOnMount)) {
+                const isAnimateInvalid =
+                    !isInitialRender.current &&
+                    currentInvalidateAnimate.current !==
+                        prevInvalidateAnimate.current
```

Comment:
If the user supplies an `invalidateAnimate`, then the animation is always invalid on the first render, since `prevInvalidateAnimate` starts off as `undefined`. So we ignore the first render.

Reply:
That's a better approach. I'll update it!